pkg/controller/migmigration/quiesce.go

```diff
@@ -12,44 +14,98 @@ import (
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// ReplicasAnnotation - annotation to set during quisce phase, which will allow to recreate the app back to it's initial state
+// in case of a migration failure or user's intent
+const (
+	SuspendAnnotation      = "migration.openshift.io/preQuiesceSuspend"
+	ReplicasAnnotation     = "migration.openshift.io/preQuiesceReplicas"
+	NodeSelectorAnnotation = "migration.openshift.io/preQuiesceNodeSelector"
+)
```

Comment:
I don't know whether stripping them from the destination is needed. We don't quiesce until all backups have been performed, so these shouldn't be showing up on the destination cluster unless someone migrates a previously-quiesced namespace in a later migration. That being said, it wouldn't hurt to look for these and delete them anyway, just in case.

Reply:
Moved annotations to the right location. With this PR such situation described by Scott would not appear that often, as the stripping is done on migration failure.